main/docker/Dockerfile

```diff
@@ -69,15 +72,18 @@ ENV APP_OPTS="/opt/metrics-portal"
 ENV ADDITIONAL_APP_OPTS=""
 
 # Build
+RUN apk add --no-cache chromium
 RUN mkdir -p /opt/metrics-portal/lib/ext && \
     mkdir -p /opt/metrics-portal/config && \
     mkdir -p /opt/metrics-portal/logs
 ADD deps /opt/metrics-portal/lib/
 ADD bin /opt/metrics-portal/bin/
 ADD config /opt/metrics-portal/config/
 ADD lib /opt/metrics-portal/lib/
+RUN chown -R mportal /opt/metrics-portal
 
 # Entry point
+USER mportal
```

Comment:
What does this do to log file permissions generated by the application when they are volume mounted out of the running container? 

Note: the behavior may be different between Docker for Mac and Docker for Linux; the former uses a VM under the covers. Testing the latter could be done by starting a VM and running the Docker image in there.